README.md

```diff
@@ -1,6 +1,8 @@
 # kuery
 
 Simple Mongo-like queries with lo-dash, at the moment the operators are very limited...
+
+### JavaScript
```

Comment:
We don't duplicated documentation for `JavaScript` _and_ `TypeScript`. Use TypeScript in the documentation.

Reply:
And we should not name the heading `TypeScript`, but `Usage` or similar.